concepts/05 Widgets/Gallery/05 Switch Between Images/10 API.md

```diff
@@ -78,19 +78,128 @@ To switch the **Gallery** to the next or previous image, call the [nextItem(anim
         // ...
     })
 
+##### Vue
+
+    <template>
+        <div>
+            <DxGallery
+                :data-source="dataSource"
+                :height="300"
+                ref="gallery"
+            />
+            <DxButton
+                text="Next"
+                @click="goToNextItem"
+            />
+            <DxButton
+                text="Previous"
+                @click="goToPreviousItem"
+            />
+        </div>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxGallery } from 'devextreme-vue/gallery';
+    import { DxButton } from 'devextreme-vue/button';
+
+    export default {
+        components: {
+            DxGallery,
+            DxButton
+        },
+        data() {
+            return {
+                dataSource: [
+                    'https://js.devexpress.com/Content/images/doc/20_1/PhoneJS/person1.png',
+                    'https://js.devexpress.com/Content/images/doc/20_1/PhoneJS/person2.png',
+                    'https://js.devexpress.com/Content/images/doc/20_1/PhoneJS/person3.png',
+                    'https://js.devexpress.com/Content/images/doc/20_1/PhoneJS/person4.png'
+                ]
+            };
+        },
+        methods: {
+            goToNextItem () {
+                this.$refs.gallery.instance.nextItem(true);
```

Comment:
In Angular, Vue, and React, it is better to use the [selectedIndex](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxGallery/Configuration/#selectedIndex) option instead.

Reply:
We decided to keep current implementation, because it's more native and compact